virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -72,6 +72,9 @@ This release adds new features and enhancements related to the following compone
 //CNV-29192 Running real-time checkup
 * You can now run a predefined xref:../../virt/monitoring/virt-running-cluster-checkups.adoc#virt-running-real-time-checkup_virt-running-cluster-checkups[checkup] to verify that your {product-title} cluster is configured to run virtualized real-time workloads.
 
+//CNV-33366 Release note: NEW - Adopt UEFI in all Windows versions
+* Windows 10 VMs now boot using UEFI with TPM.
```

Comment:
I see TPM elsewhere on the page, but does UEFI need spelling out or is it common enough?

Reply:
i think even though they're both pretty dated specifications, tpm has really only been talked about as much since the hard requirement for w11. i feel like leaving uefi as is here would be fine. thanks for checking!